java/src/main/java/com/logicalclocks/hsfs/StorageConnector.java

```diff
@@ -294,6 +294,9 @@ public String getPath(String subPath) {
     @Getter @Setter
     private String table;
 
+    @Getter @Setter
+    private String application;
```

Comment:
The application field should also be added to the map returned by the `sparkOptions()` method, similar to what you did for the Python version.